pkg/patterns/declarative/pkg/kubectlcmd/client.go

```diff
@@ -49,7 +56,8 @@ func (console) Run(c *exec.Cmd) error {
 }
 
 // Apply runs the kubectl apply with the provided manifest argument
-func (c *Client) Apply(ctx context.Context, namespace string, manifest string, validate bool, extraArgs ...string) error {
+func (c *Client) ApplyAlt(ctx context.Context, namespace string, manifest string, validate bool,
+	extraArgs ...string) error {
```

Comment:
Nit we tend not to split these over lines, or if we do we split all the args, one per line